tests/cisco_asa/show_running-config_crypto_map/cisco_asa_show_running-config_crypto_map2.yml

```diff
@@ -0,0 +1,18 @@
+---
+parsed_sample:
+  - matched_address: "CMAP_FR1"
```

Comment:
The results show the entries are getting overwritten

Reply:
@jmcgill298 thanks for catching that, i thought it might not be solvable as the last line of each block is variable.

Using continue have been able to capture blocks based on first line in each block "match address" using
`^crypto\s+map\s\S+\s\d+\s+match\s+address\s\S+\s* -> Continue.Record`